assets/javascripts/src/modules/analytics/ga.es6

```diff
@@ -16,8 +16,10 @@ const dimensions = {
     intcmp: 'dimension12'
 };
 
+const defaultTracker = 'membershipPropertyTracker';
+var gaEnabled = true;
```

Comment:
This bit I'm not sure about... I need to somehow hold a state for browsers or users that don't want to be tracked without breaking later when we call the ga function (via gaProxy)